packer/install-dependencies.sh

```diff
@@ -126,6 +126,21 @@ for NVIDIA_DEVICE_PLUGIN_VERSION in ${NVIDIA_DEVICE_PLUGIN_VERSIONS}; do
     pullContainerImage "docker" "nvidia/k8s-device-plugin:${NVIDIA_DEVICE_PLUGIN_VERSION}"
```

Comment:
is this missing a "v" for version? (I know it's not from this PR but just noticed)

Reply:
there is no "v" prefix for this image